code/game/gamemodes/cult/runes.dm

```diff
@@ -1158,7 +1158,7 @@
 		if(iscultist(C) && !C.stat)
 			users+=C
 	if(users.len>=2)
-		var/mob/living/carbon/cultist = input("Choose the one who you want to summon", "Followers of Geometer") as null|anything in (cultists - user)
+		var/mob/living/carbon/cultist = input("Choose the one who you want to summon", "Followers of Geometer") as null|anything in ((cultists - users) - user)
```

Comment:
Can't you take the `user` off of this, now, since they should be part of `users`?

Reply:
no? I don't see how `user` would get added to `users` because it's doing `orange()`

Reply:
Didn't see, though it was `range()`

Reply:
Actually, that's just as bad either way.
The user can both be in orange() and be the user.
Meaning you can actually probably summon cultists with just two people right now instead of the intended 3.